src/_tutorials/web/images/set-callback.svg

```diff
@@ -0,0 +1,3 @@
s="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe flex-start; width: 533px; height: 1px; padding-top: 35px; margin-left: 7px;"><div style="box-sizing: border-box; font-size: 0; text-align: left; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><div style="font-family: &quot;consolas&quot; , &quot;courier new&quot; , monospace ; font-size: 16px ; line-height: 22px"><div><span style="background-color: rgb(255 , 255 , 255)">..</span><span style="background-color: rgb(204 , 229 , 255)">onLoadEnd</span><span style="background-color: rgb(255 , 255 , 255)">.listen(</span><span style="background-color: rgb(204 , 229 , 255)">(e) =&gt;  requestComplete(httpRequest)</span><span style="background-color: rgb(255 , 255 , 255)">);</span></div><div style="background-color: rgb(255 , 255 , 255)">..send(<span style="color: #a31515">''</span>);</div></div></div></div></div></foreignObject><text x="7" y="39" fill="#000000" font-family="Helvetica" font-size="12px">..onLoadEnd.listen((e) =&gt;  requestComplete(httpRequest));...</text></switch></g><path d="M 60 34 L 60 104 Q 60 114 70 114 L 330 114 Q 340 114 340 104 L 340 40.37" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 340 35.12 L 343.5 42.12 L 340 40.37 L 336.5 42.12 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><rect x="110" y="90" width="190" height="50" fill="#000000" stroke="#000000" pointer-events="all" transform="translate(2,3)" opacity="0.25"/><rect x="110" y="90" width="190" height="50" fill="#ffffff" stroke="#000000" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 188px; height: 1px; padding-top: 115px; margin-left: 111px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><div>set callback function</div><div>(to call when request completes)</div></div></div></div></foreignObject><text x="205" y="119" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">set callback function...</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
Could we expand the box a bit to the right? It seems the end has some strange artifacts:

![image](https://user-images.githubusercontent.com/18372958/121576411-f7fe4380-c9ed-11eb-9445-a8c31c4deb39.png)

Also a nit: There seems to be an extra space before `requestComplete`. It's okay if you can't remove it:

![image](https://user-images.githubusercontent.com/18372958/121576593-24b25b00-c9ee-11eb-81e8-c26f4f8ca2cd.png)